packages/contentful--app-scripts/utils/constants.js

```diff
@@ -3,9 +3,14 @@ const ACCESS_TOKEN_ENV_KEY = 'CONTENTFUL_ACCESS_TOKEN';
 const ORG_ID_ENV_KEY = 'CONTENTFUL_ORG_ID';
 const APP_DEF_ENV_KEY = 'CONTENTFUL_APP_DEF_ID';
 
+const DEFAULT_BUNDLES_TO_KEEP = 50;
```

Comment:
Is there a reason we start with such a high number? Basically, is there a risk it will delete their latest active bundle? (I'm not sure about the API validations about deleting an active bundle)